website/src/data/users.tsx

```diff
@@ -1142,6 +1142,15 @@ const Users: User[] = [
     source: 'https://github.com/leon-ai/docs.getleon.ai',
     tags: ['opensource', 'versioning', 'i18n'],
   },
+  {
+    title: 'Lerna',
+    description:
+      'A fast modern build system for managing and publishing multiple JavaScript/TypeScript packages from the same repository.',
+    preview: require('./showcase/lerna.png'),
+    website: 'https://lerna.js.org',
+    source: 'https://github.com/lerna/lerna',
```

Comment:
Is the _website_ open source? I can't find it anywhere.

If not, remember to [remove the "Edit this page" links](https://docusaurus.io/docs/next/api/plugins/@docusaurus/plugin-content-docs#editUrl) at the bottom of the pages as they currently point to our repo.

Reply:
couldn't either

btw your editUrl is wrong ;)

Reply:
Thank you for the reviews! 
The edit URL is updated!

We are in the process of migrating the website to Lerna's monorepo. Here is the PR https://github.com/lerna/lerna/pull/3187, it should be done soon.

Reply:
Great—let's wait for that PR to be merged first then. No hurry.

Reply:
The PR has been merged 👍